plugins/orchestrator/src/utils/usePolling.ts

```diff
@@ -0,0 +1,116 @@
+import React, { useCallback, useRef } from 'react';
```

Comment:
There is a `hooks` directory, can we move this module there please?

Reply:
https://github.com/janus-idp/backstage-plugins/pull/1081/commits/5e918181ae1defb6a6bdb570ca5e915d32f918cf